feature-libs/product-configurator/rulebased/cpq/cpq-configurator-normalizer.ts

```diff
@@ -126,7 +126,9 @@ export class CpqConfiguratorNormalizer
       attrCode: sourceAttribute.stdAttrCode,
       name: sourceAttribute.pA_ID.toString(),
       description: sourceAttribute.description,
-      label: sourceAttribute.label,
+      label: sourceAttribute.label
+        ? sourceAttribute.label
+        : sourceAttribute.name,
```

Comment:
this does not seem to be related to 'move value from input field'. did you fix multiple issues with this PR?

Reply:
Yes. Several issues (directly or indirectly related to handling of input fields) are fixed with this pull request. All adjustments are mentioned in the pull request comment.